diff --git a/media/libstagefright/MPEG4Writer.cpp b/media/libstagefright/MPEG4Writer.cpp
index 7786c4de0c..3c02a34712 100644
--- a/media/libstagefright/MPEG4Writer.cpp
+++ b/media/libstagefright/MPEG4Writer.cpp
@@ -733,12 +733,12 @@ status_t MPEG4Writer::start(MetaData *param) {
         mIsFileSizeLimitExplicitlyRequested = true;
     }
 
-    int32_t use64BitOffset;
-    if (param &&
-        param->findInt32(kKey64BitFileOffset, &use64BitOffset) &&
-        use64BitOffset) {
+    // int32_t use64BitOffset;
+    // if (param &&
+    //     param->findInt32(kKey64BitFileOffset, &use64BitOffset) &&
+    //     use64BitOffset) {
         mUse32BitOffset = false;
-    }
+    // }
 
     if (mUse32BitOffset) {
         // Implicit 32 bit file size limit
